src/styles/blocks/main_block.scss

```diff
@@ -0,0 +1,20 @@
+.cards {
+  display: flex;
+  flex-wrap: wrap;
+  padding: $ver-padding $hor-padding;
+  width: $main-dimension;
+  margin: 0 auto;
+
+  & > * {
+    margin-right: $card-shift-right;
+    margin-bottom: $card-shift-bottom;
+  }
+
+  & > div:nth-child(4n) {
+    margin-right: 0;
+  }
+
+  & > div:nth-last-child(-n + 4) {
+    margin-bottom: 0;
+  }
```

Comment:
Use card class for this purpose, and no need to increase selector specificity

Reply:
Yes, you can't set margins for BEM-blocks, but using tags instead is a bad solution. For this cases you can use mix of block and element (your block can also be an element of it's parent block)